src/game.cpp

```diff
@@ -9087,7 +9099,7 @@ void game::reset_item_list_state(WINDOW *window, int height)
     tokens.push_back(_("<+/->Priority"));
 
     int gaps = tokens.size() + 1;
-    int letters = 0;
+    letters = 0;
```

Comment:
Not sure what you mean by that?! I needed the int before it was declared here and instead of adding a new variable I just used that one a few lines up.